media/js/base/site.js

```diff
@@ -64,10 +64,7 @@
             if (pf.indexOf("Mac") !== -1) {
                 return 'oldmac';
             }
-            if (pf === '' &&
-                    /Firefox/.test(ua) &&
-                    /Mobile|Tablet/.test(ua) &&
-                    !(/Android/.test(ua))) {
+            if (pf === '' && /Firefox/.test(ua)) {
```

Comment:
> Gave this a quick whirl and it seems to return true for me for Firefox desktop and Android UA strings?

Oops, you're right @alexgibson, my mistake. Never mind, it seems the only way to differentiate is indeed with the navigator.platform value :/